source/js/utils.js

```diff
@@ -373,7 +373,16 @@ NexT.utils = {
         script.onload = resolve;
         script.onerror = reject;
 
-        script.src = url;
+        if (typeof src === 'object') {
+          const { url, integrity } = src;
+          script.src = url;
+          if (integrity) {
+            script.integrity = integrity;
+            script.crossorigin = 'anonymous';
```

Comment:
```suggestion
            script.crossOrigin = 'anonymous';
```

As [HTMLScriptElement Interface (MDN)](https://developer.mozilla.org/en-US/docs/Web/API/HTMLScriptElement)